### DISCUSSION TITLE
Installing gpu.js

### DISCUSSION THREAD

**Discussion by k11q** [nuxt/nuxt#16216] [Questions]
Hi I need to use GPU.js on my project, and upon installing can't seem to get it working. The error states  500:Failed to fetch dynamically imported module. I already put "gpu.js" at the buildmodules in nuxt config. Can someone explain why it is not working and how to get it to work? Thank you

**Comment by TheAlexLichter** [comment]
Hey! Could you kindly provide a [reproduction](https://v3.nuxtjs.org/community/reporting-bugs/#create-a-minimal-reproduction)? 🙏

Also, GPU.js should probably not put in `buildModules` because first, `buildModules` doesn't exist in Nuxt 3 and second, it is likely not a *Nuxt module*.

**Reply by k11q** [reply]
ahh I see. So I havent even did anything yet. Just importing this on my file and already give me error, after installed with yarn. And also I think my build fail because of it so I remove the package and my build success again

 import { GPU } from 'gpu.js';
  const gpu = new GPU();
